testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -437,13 +438,26 @@ public void testMaterializedView()
                 "VALUES '" + viewName + "'");
 
         // column listing
-        checkShowColumnsForMaterializedView(viewName);
+        MaterializedResult expectedColumns = resultBuilder(getSession(), VARCHAR, VARCHAR, VARCHAR, VARCHAR)
+                .row("nationkey", "bigint", "", "")
+                .row("name", "varchar", "", "")
+                .row("regionkey", "bigint", "", "")
+                .row("comment", "varchar", "", "")
```

Comment:
Connector can return comments or something, and we're not interested in these.
you can use `QueryAssert#projected` to ignore them in assertions.

Also, the column types can vary between connectors, when eg they don't support `bigint` (like oracle)